src/index.ts

```diff
@@ -0,0 +1,17 @@
+export class Challenge {
+    num : number;
+    constructor(){
+    }
+    public buzz(i:number){
+        return new Promise(function(resolve, reject){
+            if(i%3==0 && i%5==0)
+                resolve("fizzbuzz");
```

Comment:
This is ok.
Just a consideration: remember that `resolve` and `reject` are simple callback and they don't stop the flow, any line after that will be executed. If you want to break use that in combination with `return` (i.e. `return resolve();`)